ui/src/views/api-keys/CreateTenantApiKey.tsx

```diff
@@ -11,13 +11,15 @@ import type { Tenant } from "@chirpstack/chirpstack-api-grpc-web/api/tenant_pb";
 import ApiKeyForm from "./ApiKeyForm";
 import ApiKeyToken from "./ApiKeyToken";
 import InternalStore from "../../stores/InternalStore";
+import { useTitle } from "../../stores/helpers";
 
 interface IProps {
   tenant: Tenant;
 }
 
 function CreateTenantApiKey(props: IProps) {
   const [createApiKeyResponse, setCreateApiKeyResponse] = useState<CreateApiKeyResponse | undefined>(undefined);
+  useTitle("Add", "API Keys", props.tenant.getName(), "Tenants");
```

Comment:
Sorry for be a pain, but I think it is better if this is reversed. From less specific to more specific. Then we call the `useTitle` like:

```js
useTitle("Tenants");
useTitle("Tenants", tenant.getName());
useTitle("Tenants", tenant.getName(), "Edit");

// etc...
```

I can make this update if you want (should be easy to do with find => replace regex so we don't have to do this manually).

If we want to display this as "Edit | Tenant | Tenants" etc, then we can reverse this in the `useTitle` function.

Reply:
Yes, I agree with that.